packages/driver/cypress/e2e/commands/actions/check.cy.js

```diff
@@ -616,8 +616,41 @@ describe('src/cy/commands/actions/check', () => {
         cy.on('log:added', (attrs, log) => {
           this.lastLog = log
         })
+      })
 
-        return null
+      it('can turn off logging when protocol is disabled', { protocolEnabled: false }, function () {
```

Comment:
I've been trying to be consistent about not using 'should' or 'can'. Something like this:

```suggestion
      it('turns off logging when protocol is disabled', { protocolEnabled: false }, function () {
```

Reply:
Never mind on this. I think this makes sense in this context after thinking about it.